custom_components/octopus_energy/api_client/heat_pump.py

```diff
@@ -122,6 +122,29 @@ class OctoHeatPumpControllerConfiguration(BaseModel):
     zones: List[ConfigurationZone]
 
 
+class OctoHeatPumpLivePerformance(BaseModel):
+    coefficientOfPerformance: str 
+    heatOutput: ValueAndUnit
+    powerInput: ValueAndUnit
+    outdoorTemperature: ValueAndUnit
+    readAt: str
+
+
+class OctoHeatPumpLifetimePerformance(BaseModel):
```

Comment:
issue: Please remove as this shouldn't be needed due to request to remove function in API client.